### PR TITLE
Task js-DOM-menu

### PR DESCRIPTION
**Pull request by PAtrofimov**
### task js-dom
[TASK](https://github.com/PAtrofimov/response_menu) | 
[DEMO](https://patrofimov.github.io/response_menu/)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
@PAtrofimov Would you please rename the PR so it gave an idea of what's inside right from the PRs listing?
image

**Comment by PAtrofimov** [comment]
Ok. I rename the PR. Is it allright?

File: submissions/patrofimov/js-dom/css/main.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+.header__title {
+  font-weight: 300;
+  font-size: 3em;
+  margin: 0.75em 0.25em;
+  display: inline-block;
+  color: #212121;
+}
+
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/js-dom/css/main.css]: Remove empty lines

File: submissions/patrofimov/js-dom/css/main.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ 
+}
+
+.nav__item {
+  box-sizing: border-box;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+  line-height: 24px;
+  /* padding: 20px; */
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/js-dom/css/main.css]: Remove commented code.

File: submissions/patrofimov/js-dom/css/responsive.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
+.content {
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/js-dom/css/responsive.css]: Move code from`responsive.css` to `main.css`.

File: submissions/patrofimov/js-dom/css/responsive.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.header__menu {
+  display: none;
+}
+
+
+@media screen and (max-width: 450px) {
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/js-dom/css/responsive.css]: Mobile version starts from 768px and going down.

File: submissions/patrofimov/js-dom/index.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        </a>
+      </div>
+    </header>
+	
+	<nav id="drawer" class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+		<a href="#" class="menu__link menu__link--active" data-order="1">
+    Гомер
+     </a></li><br>
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/js-dom/index.html]: Bad practice using `<br>`. Your case work without `<br>` tag.

File: submissions/patrofimov/js-dom/index.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    Мардж
+     </a></li><br>
+        <li class="nav__item"><a href="#" class="menu__link" data-order="4">
+    Лиза
+    
+  </a></li><br>
+      </ul>
+    </nav>
+	
+	 <main>
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/js-dom/index.html]: Use `#id` for scripts and `.class` for styling by css.

File: submissions/patrofimov/js-dom/js/script.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    element.classList.add(className);
+    if (textInfo) {
+        element.textContent = textInfo;
+    }
+    return element;
+};
+var createContent = function (rowdata) {
+    var content = makeElement("section", "content");
+	
+	var article = makeElement("article", "description");
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/js-dom/js/script.js]: Use Prettier for automation alignment your code.

**Comment by PAtrofimov** [comment]
Fixed everything.

**Comment by A-Ostrovnyy** [comment]
Great job!

**Pull request merged**